install_config/persistent_storage/topics/glusterfs_uninstall.adoc

```diff
@@ -8,15 +8,18 @@ ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansibl
 ----
```

Comment:
@tmorriso-rh I failed to notice this in the original PR, but the above command needs a prompt before it. So, `#`, I'd say?